purview/models/errata.py

```diff
@@ -47,3 +47,14 @@ class AdvisoryState(PurviewStructuredNode):
     name = StringProperty(required=True)
     advisory = RelationshipTo('Advisory', 'STATE_OF', cardinality=ZeroOrOne)
     creator = RelationshipTo('.user.User', 'CREATED_BY', cardinality=ZeroOrOne)
+
+
+class Product(PurviewStructuredNode):
+    """Definition of a released Product"""
+
+    associated_koji_tag = RelationshipTo('.koji.KojiTag', 'ASSOCIATED_TO')
+    associated_advisory = RelationshipFrom('Advisory', 'ASSOCIATED')
```

Comment:
This needs to be plural because more than one advisory can be associated with a product.